modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -2,7 +2,13 @@
 
 module TravelPay
   class ClaimsService
-    def get_claims(veis_token, btsss_token, params = {})
+    def initialize(travel_pay_session)
+      @session = travel_pay_session
+    end
+
+    # def get_claims(veis_token, btsss_token, params = {})
```

Comment:
I think this comment is forgotten clean-up and meant to be removed?

Reply:
yup, good catch, ty